build.gradle

```diff
@@ -242,7 +242,7 @@ project('spring-rabbit') {
 
 		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
 
-		testCompile "commons-cli:commons-cli:$commonsCliVersion"
+//		testCompile "commons-cli:commons-cli:$commonsCliVersion"
 
```

Comment:
Well, there is no comments why we need that `commons-cli`: https://github.com/spring-projects/spring-amqp/commit/1150fe48997ab6f9a612d40a51b5c6051c5e7efe#diff-c197962302397baf3a4cc36463dce5eaR230.

Any ideas ?